agent/discovery/host.go

```diff
@@ -46,16 +48,13 @@ func (h HostDiscovery) Discover() (string, error) {
 		return "", err
 	}
 
-	var si sysinfo.SysInfo
-	si.GetSysInfo()
-
 	host := hosts.DiscoveredHost{
-		OSVersion:       si.OS.Version,
+		OSVersion:       getOSVersion(),
 		HostIpAddresses: ipAddresses,
 		HostName:        h.discovery.host,
-		CPUCount:        int(si.CPU.Cpus) * int(si.CPU.Cores),
-		SocketCount:     int(si.CPU.Cpus),
-		TotalMemoryMB:   int(si.Memory.Size),
+		CPUCount:        getLogicalCPUs(),
+		SocketCount:     getPhysicalCPUs(),
```

Comment:
I wouldn't match SocketCount with the Core count. In my machine, which just has one socket (as most desktops), it returns `12` which is actually the number of cores.

Appart, from that (I know this was already there before the PR), I'd take the opportunity to rename following what other common tools do. See e.g. `lscpu` output for my machine:

```
CPU(s):                  24
  On-line CPU(s) list:   0-23
Vendor ID:               AuthenticAMD
  Model name:            AMD Ryzen 9 3900X 12-Core Processor
    CPU family:          23
    Model:               113
    Thread(s) per core:  2
    Core(s) per socket:  12
    Socket(s):           1
    Stepping:            0
    Frequency boost:     enabled
    CPU max MHz:         4672.0698
    CPU min MHz:         2200.0000
    BogoMIPS:            7586.00
```
(Not saying that we need to add all that, just that for the aspects of the CPU that we are adding, we should use standard names)

Also, I'm a fan of adding `Count` as suffix when a var reflects a sum of of something, but that's just a suggestion :-)

Reply:
Aside:

`AMD Ryzen 9 3900X 12-Core Processor` :OOOOOOO